src/Serilog.Sinks.OpenTelemetry/Sinks/OpenTelemetry/Exporters/HttpExporter.cs

```diff
@@ -59,11 +59,11 @@ public void Export(ExportLogsServiceRequest request)
     {
         var httpRequest = CreateHttpRequestMessage(request);
 
-#if NO_SYNC_HTTP_SEND
-        var response = _client.SendAsync(httpRequest).Result;
-#else
+#if FEATURE_SYNC_HTTP_SEND
         // We could consider using HttpCompletionOption.ResponseHeadersRead here.
         var response = _client.Send(httpRequest);
+#else
+        var response = _client.SendAsync(httpRequest).Result;
```

Comment:
does this need a ConfigureAwait
can there be a one liner comment over each impl variant please?
is L63 a TODO or a TOCONSIDER or should it be removed
if only netstandard2.1 wasnt here making me think!

Reply:
Done! Including clobbering of `netstandard2.1` :-)

No `ConfigureAwait()` because the code stays on the same thread, so no async continuation to run. Added a comment 👍

Reply:
Wasn't paying attention re the ConfigureAwait; comments are more than perfect now...